vm/ubpf_jit_x86_64.c

```diff
@@ -18,6 +18,8 @@
  * limitations under the License.
  */
 
+#include <linux/bpf.h>
```

Comment:
What are we pulling from linux/bpf.h? As much as possible this project tries to be platform agnostic.

Reply:
My fault!! I brought it in for the symbolic BPF register names. I added a local definition of the symbols. I hope that is okay. Thank you for the note!